### DISCUSSION TITLE
Lighthouse problem

### DISCUSSION THREAD

**Discussion by STAR-swarm** [bitcraze/discussions#1554] [Q&A]
Hi all,

We recently setup the lighthouse system for our lab. But, we noticed that on one of our base stations, the light is always blue. We also noticed that on one of the lighthouse deck, there is also only blue light. Do you know what is the issues with all these devices, just showing blue light?

**Comment by gemenerik** [comment]
Hi! Did you follow the [getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/)? If so, up to what point could you successfully follow it?

**Reply by STAR-swarm** [reply]
Hi,

Yes, I have followed and the ID was changed successfully. But the base stations always show blue light. Then another lighthouse deck on the crazyflie always show blue light even though we updated it already.

**Reply by gemenerik** [reply]
Thanks for the update! Just to clarify, it seems like the issue may be occurring at some point in the setup process, correct? Could you please specify exactly where things aren’t working as expected? The guide outlines clear steps, so there must be a point where it diverges.

I may be misunderstanding something here, but knowing where it stops working will really help pinpoint the issue.

**Reply by STAR-swarm** [reply]
I followed the steps, all seems well. But it appears that only one of the base station only show blue light, the others shows white light and can be detected by the lighthouse decks. It appears that the base station may be faulty, as it cannot be detected by the lighthouse deck?

**Reply by gemenerik** [reply]
All seems well, so [you are also receiving the calibration data](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/#wait-for-calibration-of-the-base-stations) from the base station with the blue light?

**Reply by STAR-swarm** [reply]
The lighthouse deck did not receive any data from the base station with blue light.

**Reply by gemenerik** [reply]
It seems like the system is stuck in calibration or initialization. Ensure that the basestation is securely mounted in a very stable position. Carefully follow the ID setup process, as any misstep could disrupt calibration. If you take a photo of the basestation while it’s on, do you see any purple lights similar to those in [this picture](https://www.bitcraze.io/images/tutorials/getting_started_with_lighthouse/two_basestations_one_crazyflie.jpg)?

**Reply by STAR-swarm** [reply]
![base_station](https://github.com/user-attachments/assets/feefceec-1deb-437e-a81c-cb90a50afd14)
I actually only see a blue light and no other lights.

**Reply by gemenerik** [reply]
It looks like it's broken. Please send us an e-mail at support@bitcraze.io
